src/dashboard/ChartDashboardObject.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace luya\admin\dashboard;
+
+use Yii;
+use yii\helpers\Html;
+use yii\helpers\ArrayHelper;
+
+/**
+ * Fast generated Dashboard Objects.
+ *
+ * The default object is the default class for all {{luya\admin\base\Module::$dashboardObjects}} items without a class defintion.
+ *
+ * @author Oom <baqianxin@163.com>
+ * @since 1.0.2
+ * @example:
+ * ```
+ *     public $dashboardObjects = [
+ *         [
+ *           'class' => 'luya\admin\dashboard\ChartDashboardObject',
+ *           'dataApiUrl' => 'admin/api-addressbook-contact/sale',
+ *           'title' => 'Echart'
+ *         ]
+ *     ];
+ *
+ *```
+ */
+class ChartDashboardObject extends BaseDashboardObject
+{
+
+
+  /**
+   * @var array Options to generate the wrapper element. Generates a tag like:
+   *
+   * ```
+   * <div class="card-panel" ng-controller="DefaultDashboardObjectController" ng-init="loadData(\'{{dataApiUrl}}\');">
+   *     <!-- content from: $outerTemplate -->
+   * </div>
+   * ```
+   *
+   * The tag element `<div>` can be changed by overriding the key `tag`.
+   */
+    public $wrapperOptions = [
+        'class' => 'card',
+        'tag' => 'div'
+    ];
+
+    /**
+     * @var string The wrapper template which is by default:
+     * @var string $uniqid The echarts's id : id="charts_$uniqid"
```

Comment:
its a method - not a variable.

Please read carefully the guide section again about how we phpdoc on methods, its all described and written. you will see this on every class in luya.

https://github.com/luyadev/luya/blob/master/docs/guide/luya-guideline.md#phpdoc